streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/schedule/TaskAssignerImpl.java

```diff
@@ -37,11 +36,12 @@ public ExecutionGraph assign(JobGraph jobGraph, List<RayActor<JobWorker>> worker
       executionNode.setNodeType(jobVertex.getVertexType());
       List<ExecutionTask> vertexTasks = new ArrayList<>();
       for (int taskIndex = 0; taskIndex < jobVertex.getParallelism(); taskIndex++) {
-        vertexTasks.add(new ExecutionTask(taskId, taskIndex, workers.get(taskId)));
+        vertexTasks.add(new ExecutionTask(taskId, taskIndex, createWorker(jobVertex)));
         taskId++;
       }
       executionNode.setExecutionTasks(vertexTasks);
       executionNode.setStreamOperator(jobVertex.getStreamOperator());
+      executionNode.setLanguage(jobVertex.getLanguage());
```

Comment:
remove the setLanguage method from the executionNode and calculate the language from the operator.

Reply:
fixed